src/Tizen.NUI.Components/Controls/ItemsView/CollectionView.cs

```diff
@@ -0,0 +1,815 @@
+            {
+                return _itemsLayouter;
+            }
+            set
+            {
+                _itemsLayouter = value;
+                base.ItemsLayouter = value;
+                if (value == null)
+                {
+                    NeedInitalizeLayouter = false;
+                    return;
+                }
+
+                if ((_itemsSource != null) && (_itemsTemplate != null))
+                {
+                    if (InternalItemSource == null)
+                    {
+                        InternalItemSource = ItemsSourceFactory.Create(this, _itemsLayouter);
+                    }
+
+                    _itemsLayouter.Initialize(this);
+                    _itemsLayouter.RequestLayout(0.0f, true);
+                    if (ScrollingDirection == Direction.Horizontal)
+                    {
+                        ContentContainer.SizeWidth =
+                            _itemsLayouter.CalculateLayoutOrientationSize();
+                    }
+                    else
+                    {
+                        ContentContainer.SizeHeight =
+                            _itemsLayouter.CalculateLayoutOrientationSize();
+                    }
+                    NeedInitalizeLayouter = false;
+                }
+                else NeedInitalizeLayouter = true;
+            }
+        }
+
+        /// <summary>
+        ///  DataTemplate of group header. Group feature is not supported yet.
+        /// </summary>
+        /// <since_tizen> 6.5 </since_tizen>
+        /// This may be public opened in tizen_6.0 after ACR done. Before ACR, need to be hidden as inhouse API
+        [EditorBrowsable(EditorBrowsableState.Never)]
+		public DataTemplate GroupHeaderTemplate;
+
+        /// <summary>
+        /// DataTemplate of group footer. Group feature is not supported yet.
+        /// </summary>
+        /// <since_tizen> 6.5 </since_tizen>
+        /// This may be public opened in tizen_6.0 after ACR done. Before ACR, need to be hidden as inhouse API
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public DataTemplate GroupFooterTemplate; 
+
+        /// <summary>
+        /// Binding Property of selected item in single selection.
+        /// </summary>
+        /// <since_tizen> 6.5 </since_tizen>
+        /// This may be public opened in tizen_6.0 after ACR done. Before ACR, need to be hidden as inhouse API
+        [EditorBrowsable(EditorBrowsableState.Never)]
+		public static readonly BindableProperty SelectedItemProperty =
+			BindableProperty.Create(nameof(SelectedItem), typeof(object), typeof(CollectionView), null,
+				propertyChanged: (bindable, oldValue, newValue)=>
+                {
+			        var colView = (CollectionView)bindable;
+                    var args = new SelectionChangedEventArgs(oldValue, newValue);
+                    oldValue = colView.selectedItem;
+                    
+                    foreach(View item in colView.ContentContainer.Children)
```

Comment:
`foreach`  뒤에 공백 1개 넣어주세요.
```suggestion
                    foreach(View item in colView.ContentContainer.Children)
                    foreach (View item in colView.ContentContainer.Children)
```